Faultify.Tests/UnitTests/ConstantTests.cs

```diff
@@ -54,11 +54,11 @@ public void BooleanConstant_PostMutation_TrueToFalse()
 
             // Act
             byte[] mutatedBinary =
-                DllTestHelper.MutateField<ConstantAnalyzer>(binary, ConstantBoolTrueName, false);
+                DllTestHelper.MutateField<ConstantAnalyzer>(binary, ConstantBoolTrueName, 0);
```

Comment:
why? isn't `false` more semantic?

Reply:
I think these would be better as constants rather than magic literals

Reply:
Again, it's a typing issue. Faultify keeps tossing around raw `object`s during conversion and generation, then suddenly casting them into `int`s for bytecode purposes. This test case kept failing because the method it was testing did everything exactly as specified, but it would convert booleans into ones and zeros (which I assume is again for bytecode purposes).

Reply:
Also, FYI, 0 and 1 (and also -1 and 2) aren't magic literals.

Reply:
I would say it depends on the context. In an OO language, using 1 and 0 as booleans should be stated more clearly. Especially since given the context, I have no idea that the 0 there is supposed to represent falsehood and not something else.

Reply:
That's a fair point. You reckon a comment would suffice? I feel like making a constant in the vein of `BooleanFalse = 0` just sounds redundant and would make future developers think they should just shove a boolean in.

Reply:
I mean, I think a constant definition in the class would be the way to go, but I guess a comment is a fair compromise. Lemme see if I can do it directly through github

```suggestion
                DllTestHelper.MutateField<ConstantAnalyzer>(binary, ConstantBoolTrueName, 0 /* False */);
```

Reply:
I'll let you commit it if you think it works